packages/core-mobile/package.json

```diff
@@ -305,5 +305,9 @@
     "stream": "stream-browserify",
     "vm": "vm-browserify",
     "zlib": "browserify-zlib"
+  },
+  "volta": {
+    "node": "18.14.1",
+    "yarn": "1.22.10"
```

Comment:
yarn will need to be >= 3.6.4. also, can you add this for all the package.json in the repo?